packages/forms/resources/css/components/select.css

```
@@ -27,6 +27,10 @@ select:not(.choices) {
     background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 20 20'%3E%3Cpath stroke='%236b7280' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='m6 8 4 4 4-4'/%3E%3C/svg%3E");
     background-position: right 0.5rem center;
     background-size: 1.5em 1.5em;
+
+    &:has('.choices__button') {
+        @apply pr-14;
```

```suggestion
        @apply pe-14;
```

Please also compile the assets with `npm run build`